### ISSUE TITLE
Content is wrong for WS2019/HCI

### ISSUE THREAD

**Issue by carlosmayol** [MicrosoftDocs/windowsserverdocs#5447]
The captures, videos and text info is referring to old properties (Type) or (Parent) while newer OS versions those are: 
FaultDomainType and Faultdomain respectively

also the article misses information of how  the AD Sites and Services are leveraged when "AutoAssignNodes" property is set to 1

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a03180ce-ce5e-a242-534e-a6d2a71fddc3
* Version Independent ID: 009bf11f-5419-1da4-36eb-89f16bf82ca2
* Content: [Fault domain awareness](https://docs.microsoft.com/en-us/windows-server/failover-clustering/fault-domains)
* Content Source: [WindowsServerDocs/failover-clustering/fault-domains.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/failover-clustering/fault-domains.md)
* Product: **windows-server**
* Technology: **failover-clustering**
* GitHub Login: @cosmosdarwin
* Microsoft Alias: **cosdar**

**Comment by illfated** [comment]
Good point. Would you be able to contribute via Pull Request to have those documents updated, or are you already in a dialog with the Microsoft Docs Server Team regarding this issue?

As far as I am aware, members of the Microsoft organization here on GitHub are often able to access the private Microsoft repository, which also gives access to creating internal Pull Requests.

**Comment by illfated** [comment]
Thank you for the @ mentions, I also hope one of them may have time to look into this issue. Otherwise, anyone with access to a newer server setup relevant to this article document are welcome to update it with relevant graphics.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
